pkg/commands/builder/status.go

```diff
@@ -134,6 +134,23 @@ func printBuilderReadyStatus(bldr *v1alpha2.Builder, writer io.Writer) error {
 		return err
 	}
 
+	cpTableWriter, err := commands.NewTableWriter(writer, "BuildpackName", "     BuildpackKind")
+	  if err != nil {
+	  	return nil
+	  }
+
+	for _, entry := range bldr.Spec.Order {
+		for _, ref := range entry.Group {
+			if ref.ObjectReference.Name != "" && ref.ObjectReference.Kind != "" {
+			err := cpTableWriter.AddRow(ref.ObjectReference.Name, ref.ObjectReference.Kind)
+		if err != nil {
+			return err
+		}
+	}
+	 }
+	 cpTableWriter.Write()
+	}
```

Comment:
Can you run `gofmt` on your files? I think there's some weird whitespace padding in here

Reply:
Done